web/lattice_main/views.py

```diff
@@ -51,20 +52,28 @@ def view_compiled(request):
     }
     return render(request,"lattice_main/lattice_view.html",context)
 
-# TODO restore this for debugging
-# def render_to_file(request,slices, output_file_name): # (slices : List[patches.Lattice], output_file_name : str) -> str
-#     slices = list(map(sparse_lattice_to_array, slices))
-#     styles_map = styles_map
-#     patches = patches
+# def render_to_file(request) -> str:
```

Comment:
I suppose this commented out code could go now?

Reply:
Indeed. I suppose I can always dig it up mining old commits